### DISCUSSION TITLE
Conversations export

### DISCUSSION THREAD

**Discussion by WilliamDrmrd** [danny-avila/LibreChat#2995] [Q&A] [answered]
### What features would you like to see added?

Hello I can’t seem to find a way to get a JSON out of every conversations I’ve had, is there a way or is it a feature not yet there ? 

Thanks in advance, love the work put on this project

### More details

.

### Which components are impacted by your request?

General

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There is no native feature to export all conversations by all users at once, but if that's ever needed for whatever reason, you can use mongo export features, in particular, mongodump.

**Reply by WilliamDrmrd** [reply]
Would be a convenient adding in the UI are you considering implementing it in the future ?

**Reply by danny-avila** [reply, marked as answer]
There's an open PR: https://github.com/danny-avila/LibreChat/pull/2701

Looks like @jakubmieszczak is making progress on it 👍
